test/unit/createOauthClient.test.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import createOauthClient, { OauthClient } from '../../src/createOauthClient'
+import { oauthConfig } from './test-config'
+
+describe('createOauthClient', (): void => {
+  test('returns an object when using a valid config', (): void => {
```

Comment:
when writing test for `createStorageModule` I wrote tests with the `it('should ...')` syntax rather than the `test('...')` one. I figure we should stick to one :) Any preference?